### ISSUE TITLE
What are the plans for the CMS?

### ISSUE THREAD

**Issue by boscobass** [noumo/easyii#130]
there is some new functionality plan?
or some adjustment?

**Comment by noumo** [comment]
planning to release this 
https://github.com/noumo/easyii/compare/master...dev?expand=1

**Comment by boscobass** [comment]
something i can help?

**Comment by noumo** [comment]
Yes, now im working on new docs, help with text correction(my english is far from perfect) would be great.

**Comment by boscobass** [comment]
@marcelodeandrade can help too

**Comment by boscobass** [comment]
there any way to contribute code?

**Comment by noumo** [comment]
if you find any issue, you can always to create pull requests with fixes

**Comment by boehsermoe** [comment]
There is still a opened pull request #99
When do you think you can merge it? I did already used it for some own projects/website and it looks really cool!

**Comment by noumo** [comment]
@boehsermoe 
Your job is awesome.
I thought a lot about your Content module and decide to refactor Page module, you can look in current dev branch. Now pages have tree view structure and every one has fields.

**Comment by marcelodeandrade** [comment]
Of course. I will see something to do
